rollouts/controllers/rollout_controller.go

```diff
@@ -414,7 +499,9 @@ func (r *RolloutReconciler) rolloutTargets(ctx context.Context, rollout *gitopsv
 		})
 	}
 
-	return clusterStatuses, nil
+	waveComplete := concurrentUpdates > 0
```

Comment:
not clear... it only means the waveIsInProgress...right ?

Reply:
Good catch. Updated the variable name in the latest commit.